RELEASE_NOTES.md

```diff
@@ -24,6 +24,8 @@
 
 - NaN values are treated as missing when gaps are determined in the `OrderedRingBuffer`.
 
+- A new class `format_formula` has been added to format formulas from `FormulaEngine`. For convenience, `FormulaEngine` now provides an `__str__` override that makes use of this to make formulas more intuitively introspectible.
+
```

Comment:
I wouldn't enter into that much detail, for the user the only visible change is when printing formulas the output was improved to use infix notation. For example something like:

```suggestion
- Now when printing `FormulaEngine` for debugging purposes the the formula will be shown in infix notation, which should be easier to read.
```